### DISCUSSION TITLE
v2.16.6

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#492] [General]
Fixes and improved performance.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.16.5...v2.16.6

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.6'>v2.16.6</a>.</em>

**Comment by kissbzm** [comment]
大神可以帮忙添加下 快捷键吗
2.4ghz频段 和 5.0ghz频段 快速切换吗
Tasker  进行蓝牙连接后 自动切换 2.4ghz频段 和 5.0ghz频段 
因为 有些设备 需要2.4GHZ频段 有些 5.0ghz频段

Can Da Shen help add shortcut keys
Can I quickly switch between the 2.4ghz and 5.0ghz frequency bands
Tasker automatically switches between 2.4GHz and 5.0GHz frequency bands after Bluetooth connection
Because some devices require a 2.4GHz frequency band, while others require a 5.0GHz frequency band

**Comment by kissbzm** [comment]
![微信图片_20230818124429](https://github.com/Mygod/VPNHotspot/assets/49543087/1829cabd-f308-4cf1-bc39-aa36dcb30d04)

类似这样 选着2.4ghz 或者5.0ghz 启动
Similar to this, select 2.4ghz or 5.0ghz to start

**Reply by Mygod** [reply]
A PR is welcome. :)

**Reply by kissbzm** [reply]
可以添加 Tasker 支持 2.4ghz 5.0ghz 插件吗
Can Tasker support 2.4ghz 5.0ghz plugins be added
